benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -2,47 +2,69 @@
 from abc import ABC, abstractmethod
 
 import torch._C._instruction_counter as i_counter
+import torch._dynamo.config as config
+from torch._dynamo.utils import CompileTimeInstructionCounter
 
 
 class BenchmarkBase(ABC):
     _instruction_count = False
+    _compile_time_instruction_count = False
 
     def enable_instruction_count(self):
         self._instruction_count = True
         return self
 
+    def enable_compile_time_instruction_count(self):
+        self._compile_time_instruction_count = True
+        return self
```

Comment:
At this point I think it's good to actually add docs to all of these. In particular, it seems good to carefully define what compile time instruction count means, because, e.g., if I do a forward-backward compile, I'm pretty sure your current formulation doesn't capture backwards compilation.

Reply:
i will be very interesting if it does not because that means we do not capture that in our c_profiles nor out strobelight profiler, because   @maybe_cprofile is only wrapping _compile_inner

Reply:
I will add a documentation though

Reply:
is there a specific entry point for the missing compilation part that i can wrap?